cmd/ejson/main.go

```diff
@@ -65,9 +66,22 @@ func main() {
 					Name:  "o",
 					Usage: "print output to the provided file, rather than stdout",
 				},
+				cli.BoolTFlag{
+					Name:  "key-from-stdin",
+					Usage: "Read the private key from STDIN",
+				},
 			},
 			Action: func(c *cli.Context) {
-				if err := decryptAction(c.Args(), c.GlobalString("keydir"), c.String("o")); err != nil {
+				var userSuppliedPrivateKey string
+				if c.Bool("key-from-stdin") {
```

Comment:
@burke would you prefer a `--key-from-stdin` flag or the approach outlined above without a flag?